src/frontend/screens/Quests/components/QuestDetailsViewPlay/index.tsx

```diff
@@ -233,6 +233,7 @@ export const QuestDetailsViewPlayWrapper = observer(
             steamAccountLinked: false
           },
           playStreak: getPlaystreakArgsFromQuestData({
+            // @ts-expect-error - fix these types later
```

Comment:
Not related, needed to add this here after updating the `utils` package since the types were not matching.

Reply:
I will address this today.